loom/common/core/allocator.h

```diff
@@ -272,20 +212,20 @@ struct loom_allocator
 * operators, if you want an array use the templated vector class.
 *
 ************************************************************************/
-#define lmNew(allocator)                new(allocator, __FILE__, __LINE__)
+#define lmNew(allocator)                new(allocator, __FILE__, __LINE__, 1, 2, 3)
```

Comment:
No they are just to distinguish the symbol from others so users link with our new. We could do one weird Loom-specific type instead. ie an arg that is `const LS::NewLock&` or something so there's no risk at all of collision.